engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/views/docker-tags.html

```diff
@@ -32,6 +33,9 @@ <h2 translate>Docker Tags</h2>
             </a>
           </td>
           <td bst-table-cell>
+            {{ tag.manifest.schema_version }}
+          </td>
+          <td bst-table-cell>
```

Comment:
Yeah I added a diff issue for that page -> http://projects.theforeman.org/issues/19473